packages/jest-matchers/src/matchers.js

```diff
@@ -91,6 +91,32 @@ const matchers: MatchersObject = {
     return {message, pass};
   },
 
+  toChange(received: Function, expected: Function) {
```

Comment:
> I think you might want to put some protection around these variables to ensure they are functions
I agree.
```js
if (typeof received !== 'function') {
 // WARN THEM!
}

if (typeof expected !== 'function') {
  // WARN THEM!
}
```

a simple addition to it that could work. Is there another and prettier way of doing so?